cli/build.rs

```diff
@@ -6,6 +6,7 @@ use std::{
     time::SystemTime,
 };
 
+#[allow(dead_code)]
```

Comment:
why?

Reply:
CI fails otherwise.

Reply:
That's because of the build script import - don't import it, just copy paste the function